### DISCUSSION TITLE
Looking for similar "Citizen Science" style Pi projects

### DISCUSSION THREAD

**Discussion by nbanderso** [mcguirepr89/BirdNET-Pi#814] [General]
Hi all,

For the past year I have been absolutely loving BirdNET-Pi.  In fact, prior to setting this up, I had virtually zero interest in the local bird population and today birding is one of my favorite hobbies.  I really appreciate @mcguirepr89 and the community for turning me on to such a wonderful part of life!  There is something about this avenue of learning and exploring that really appeals to me.

With that being said, I am curious if there are other similar style projects are out there.  Is anyone doing something like this for a different subject matter?  

I'm hoping we can come up with some sort of running list so that others like me can do some dabbling.  Here is what I have found so far:

- [Raspberry Shake](https://raspberryshake.org/) allows you to monitor for earthquakes and ground shaking from within your home.

**Comment by lloydbayley** [comment]
I've just started a [PiAware](https://flightaware.com/adsb/piaware/) system. Just needed the USB SDR which is tuned for the proper bandwith and was able to build my own antenna. It's worth a look AND you acutally help out by reporting your positional data back to the mothership :alien:, I mean main FlightAware website! :)

I also have a Pi 3B+ running the [ARM version of Folding at Home](https://downey.io/blog/folding-at-home-raspberry-pi-arm/) whilst it's not being used. If I need it for prototyping, I just shut it down, bung in a new SD card and go from there. When I'm finished, I pop the FAH card back in and plug it back in.

Doing my bit where I can.
